app/src/main/java/com/hover/stax/channels/Channel.java

```diff
@@ -97,6 +97,16 @@ public Channel(JSONObject jsonObject, String rootUrl) {
         return sorted_list;
     }
 
+    public static List<Channel> filter(List<Channel> channels, String searchValue) {
```

Comment:
Preferably let's keep models clean. Logic like this can be moved to a ViewModel or where it's being used.

Reply:
Agreed. Even the toFilterableString method below doesn't belong here. I think the root of the problem is the ChannelViewModel doing too much. Possibly need a specialized channel search view model to reduce duplication properly

Reply:
Done